docs/src/documentation/03-components/09-text/heading/03-accessibility.mdx

```diff
@@ -34,8 +34,12 @@ renders:
 <div>Hello World!</div>
 ```
 
-If the `as` prop is set to `"div"` (or undefined), the `role` prop is optional, but only accepts one possible value (if not `undefined`): `"heading"`.
-If the `role` prop is set to `"heading"`, the `level` prop must be defined as well. It will tell assistive technologies the level of the heading.
+It is not semantically wrong but won't tell screen readers that the element is a heading. This should be used only for decorative purposes.
+
+Therefore, it's highly recommended to use the `role="heading` to indicate to assistive technologies that this element should be treated and formatted like a heading.
```

Comment:
This is not necessarily true. Sometimes we don't want to recommend heading. The `Heading` component can be used as a decorative element. It is better to not use headings than using them with incorrect semantics.

I would refrain from giving the recommendation. We can specify what each prop does and the effects and consequences, but abstain from recommending

Reply:
Yes, but if it's not decorative purposes, then it is recommended, no? If it's not possible to use other `as` than `div` for some reason and it's not decorative heading, then they should use it, no?

Reply:
Yes, yes. My only concern is with the term `highly recommended`. I would personally feel pressured to find a reason to use it, and eventually end up using it wrong 😄